packages/flutter/test/widgets/scrollable_helpers_test.dart

```diff
@@ -590,4 +606,77 @@ void main() {
     expect(find.text('The cape as red as blood'), findsOneWidget);
     expect(find.text('The hair as yellow as corn'), findsOneWidget);
   });
+
+  testWidgets('Invoking a ScrollAction when notificationContext is null does not cause an exception.', (WidgetTester tester) async {
```

Comment:
```suggestion

  // Regression test for https://github.com/flutter/flutter/issues/158063.
  testWidgets('Invoking a ScrollAction when notificationContext is null does not cause an exception.', (WidgetTester tester) async {
```

I went ahead and made an issue so we can link it here:
- https://github.com/flutter/flutter/issues/158063